src/view/viewTabs.component.tsx

```diff
@@ -3,19 +3,12 @@ import Tab from '@mui/material/Tab';
 import Tabs from '@mui/material/Tabs';
 import { styled } from '@mui/material/styles';
 import React from 'react';
-import { Route, Routes, useLocation, useNavigate } from 'react-router-dom';
+import { Outlet, useLocation, useNavigate } from 'react-router-dom';
 import { TAB_VALUES, TabValue } from '../app.types';
-import Catalogue from '../catalogue/catalogue.component';
-import CatalogueItemsLandingPage from '../catalogue/items/catalogueItemsLandingPage.component';
-import Items from '../items/items.component';
-import ItemsLandingPage from '../items/itemsLandingPage.component';
-import Manufacturer from '../manufacturer/manufacturer.component';
-import ManufacturerLandingPage from '../manufacturer/manufacturerLandingPage.component';
-import Systems from '../systems/systems.component';
 import { getSciGatewayPageHeightCalc, isRunningInDevelopment } from '../utils';
-import { HomePage } from '../homePage/homePage.component';
 
 export const paths = {
+  any: '*',
```

Comment:
Could this also be moved to `App.tsx` and then be used inside the router itself.

Reply:
@MatteoGuarnaccia5 I think you missed out this.

Reply:
I did add it to `App.tsx`, this any was just for `vewTabs.component.test.tsx` as it doesn't work with any other of the paths

Reply:
Ah sorry, I meant the whole `paths` object so they are imported from App.tsx and used inside App.tsx inside the router for the paths.